OnOff.ps1

```diff
@@ -0,0 +1,78 @@
+# PowerShell Script to Automate Chatflow On and Off

```

Comment:
Please do the following:
- move this into a folder called `scripts`
- rename it to `windows-utils.ps1`
- we need a set-up step. Most of the actions will be done only once
- Update the readme with the instructions

Reply:
done